survivor_merge_helper.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -o pipefail
+set -o errexit
+
```

Comment:
I'd add `set -o nounset` just as an extra precaution against one of the 10 parameters not being passed.
```suggestion
set -o nounset

```

Reply:
added!